eng/pipelines/cadl-ranch-typespec.yml

```diff
@@ -18,5 +18,5 @@ steps:
       azureSubscription: "Cadl Ranch Storage"
       scriptType: "bash"
       scriptLocation: "inlineScript"
-      inlineScript: cadl-ranch upload-coverage --generatorName go --storageAccountName azuresdkcadlranch --generatorVersion $(node -p -e "require('$(System.DefaultWorkingDirectory)/packages/typespec-go/package.json').version")
+      inlineScript: cadl-ranch upload-coverage --generatorName go --storageAccountName azuresdkcadlranch --generatorVersion $(node -p -e "require('$(System.DefaultWorkingDirectory)/packages/typespec-go/package.json').version") --generatorMode standard
```

Comment:
```suggestion
      inlineScript: cadl-ranch upload-coverage --generatorName go --storageAccountName azuresdkcadlranch --generatorVersion $(node -p -e "require('$(System.DefaultWorkingDirectory)/packages/typespec-go/package.json').version") --generatorMode azure
```

Reply:
@timotheeguerin do you know the impact of changing from `standard` to `azure` here?

Reply:
https://azure.github.io/cadl-ranch/

This populate this dashboard

Azure mode supposed to be when you understand all the azure extra functionality.

Reply:
I see. I ask because Python is specifying `standard` so I was just following suit. I do see they have results for standard and Azure on the dashboard though.

Reply:
yeah not sure, I am not up to date on the status of the azure vs standard emitter and what you should be doing.

@lmazuel are emitters supposed to have both emitters mode today?

Reply:
Azure mode is for branding and standard mode is for non-branding SDK generation.